URL type regex hyphen bug (#23120)
demisto/content: Packs/CommonTypes/ReleaseNotes/3_3_38.md

#### Indicator Types
- **urlRep**

> - Fixed an issue in the URL type regex that caused it to not catch the full domain when it was seperated by a hyphen.

Reviewer: ```suggestion
- Fixed an issue where the URL type regex did not catch the full domain indicator when it was separated by a hyphen.
```
